### PR TITLE
Change header font to Chicago, self-host fonts

### PR DESCRIPTION
**Pull request by jberdine**

### CODE REVIEW

File: www/src/css/custom.css [hunk 1]
```diff
@@ -4,7 +4,23 @@
  * work well for content-centric websites.
  */
 
-@import url("https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&family=Space+Mono&display=swap");
```
Comment (jberdine) [inline www/src/css/custom.css:7]: This seems to work fine, but lets google track all traffic to our docs site.

**Comment by jberdine** [comment]
image

File: www/src/css/custom.css (hunk 1 above)
Comment (pikatchu) [reply www/src/css/custom.css:7]: I think that's ok

**Comment by pikatchu** [comment]
Ship it!

**Review by pikatchu — APPROVED** [review]
Awesome thanks!

**Pull request merged**